src/components/MovieDetails/MovieDetails.js

```diff
@@ -44,7 +44,7 @@ class MovieDetails extends Component {
 
   displayUserRatingConditional = () => {
     if(this.props.userInfo.id){
-      return `Your rating: ${this.showUserRating()}`
+      return this.showUserRating();
```

Comment:
I broke this text out from the function and added to the component so I could add in a span tag around the user rating. See changes below.

Reply:
Great idea!